README.md

```diff
@@ -4,11 +4,11 @@ The following [.NET releases](./releases.md) are currently supported:
 
 |  Version  | Release Date | Support | Latest Patch Version | End of Support |
 | :-- | :-- | :-- | :-- | :-- |
-| [.NET 8](release-notes/8.0/README.md) | [November 8, 2023](https://devblogs.microsoft.com/dotnet/announcing-dotnet-8-preview-4/) | [LTS][policies] | [8.0.0-preview.7][8.0.0-preview.7] | November 10, 2026 |
+| [.NET 8](release-notes/8.0/README.md) | [November 8, 2023](https://devblogs.microsoft.com/dotnet/announcing-dotnet-8-preview-7/) | [LTS][policies] | [8.0.0-preview.7][8.0.0-preview.7] | November 10, 2026 |
```

Comment:
```suggestion
| [.NET 8](release-notes/8.0/README.md) | [November 2023](https://devblogs.microsoft.com/dotnet/announcing-dotnet-8-preview-7/) | [LTS][policies] | [8.0.0-preview.7][8.0.0-preview.7] | November 10, 2026 |
```

Reply:
I think we can reasonably write 11/14 per https://www.dotnetconf.net/

Reply:
I wasn't sure about that. Let me update it.